datafusion/sqllogictest/test_files/array.slt

```diff
@@ -1044,6 +1044,25 @@ select make_array(['a','b'], null);
 ----
 [[a, b], ]
 
+## array_sort (aliases: `list_sort`)
+query ???
+select array_sort(make_array(1, 3, null, 5, NULL, -5)), array_sort(make_array(1, 3, null, 2), 'ASC'), array_sort(make_array(1, 3, null, 2), 'desc', 'NULLS FIRST');
+----
+[, , -5, 1, 3, 5] [, 1, 2, 3] [, 3, 2, 1]
+
+query ?
+select array_sort(make_array(a,d,e), 'DESC', 'NULLS LAST') from values;
```

Comment:
Should  `from values`  be removed?

Reply:
If you intend to test on table. You can do `select array_sort(column1, 'DESC', 'NULLS LAST') from values;`